src/frontend/src/components/AnswerItem.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="answer-item-box">
+    <div>작성자: {{ fetchedAnswer.userId }}</div>
+    <div class="answer-temp">
+      <div class="answer-content">
+        <p class="answer-content-value" v-if="!this.updateEditFlag">
+          {{ fetchedAnswer.content }}
+        </p>
+        <v-textarea
+          outlined
+          v-else
+          v-model="fetchedAnswer.content"
+        ></v-textarea>
+      </div>
+      <v-btn v-if="this.updateEditFlag" @click="update">
+        수정 확인
+      </v-btn>
+      <v-btn v-else @click="updateBtnHandler">수정</v-btn>
+      <v-btn @click="deleteBtnHandler">삭제</v-btn>
+    </div>
+    <hr class="answer-line" />
+  </div>
+</template>
+
+<script>
+export default {
+  computed: {
+    fetchedAnswer() {
+      return this.$store.getters.fetchedAnswer(this.answerId);
+    }
+  },
+  props: {
+    answerId: Number
+  },
+  data: function() {
+    return {
+      updateEditFlag: false
+    };
+  },
+  methods: {
+    deleteBtnHandler: function() {
+      this.$store.dispatch("DELETE_ANSWER", this.answerId);
+      console.log(this.$store.getters.fetchedAnswers);
+    },
+    updateBtnHandler: function() {
+      if (this.updateEditFlag === true) {
+        this.update();
+      }
+      this.updateEditFlag = !this.updateEditFlag;
+    },
```

Comment:
```suggestion
    updateBtnHandler: function() {
      this.updateEditFlag = !this.updateEditFlag;
    },
```
이 부분은 없어도 되지 않을까요?
updateEditFlag는 디폴트로 false이므로 수정 버튼을 누르면 updateEditFlag만 true로 바꿔주는 역할만 하면 될 것 같은데, 이 부분을 추가해주신 이유가 있을까요?

Reply:
기존에 버튼으 설계가 지금과 다를 때 코드인데
수정을 안했네요 삭제했습니다